src/PetProjects.Transactions.Contracts/Topics/V1/Commands/TransactionCommands.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace PetProjects.Transactions.Contracts.Topics.V1.Commands
+{
+    using System;
+    using Framework.Kafka.Contracts.Topics;
```

Comment:
The namespace of the kafka framework should be PetProjects.Framework.Kafka.
Besides that, Why do we have a Kafka.Contracts.Topics namespace in the framework? We're going to have two "Contracts" namespaces in two different repos?

Reply:
The name here isn't the best. 

We have a contracts in framework.kafka in order to avoid having to install the full kafka framework when we are only defining messaging contracts.

The name Contracts should/could change in Framework.Kafka to only Topics.